sdk/loadtestservice/Azure.Developer.LoadTesting/README.md

```diff
@@ -0,0 +1,101 @@
+# Azure LoadTesting client library for .NET
```

Comment:
@NiveditJain can you please add a space and make it "Azure Load Testing"

Reply:
updated!